docker-ci/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Parent image for zingolibs CI
+FROM parent-ci:latest
+
+# RUN git clone https://github.com/zingolabs/zingolib.git
+RUN git clone https://github.com/Oscar-Pepper/zingolib.git # test only
+
+# Copy zcash and lightwallet binaries into zingolib regtest directory
+WORKDIR zingolib
+RUN git checkout ci-update # test only
+RUN cp /usr/bin/lightwalletd /usr/bin/zcashd /usr/bin/zcash-cli ./regtest/bin/
```

Comment:
Can we use `ln -s` instead of `cp`?

Reply:
I suspect `ln -s` will  be more efficient/fast.

Reply:
added in #221